.github/workflows/benchmark.yml

```diff
@@ -186,7 +251,7 @@ jobs:
         # To keep the directory structure and file permissions, tar them
         # See also https://github.com/actions/upload-artifact#maintaining-file-permissions-and-case-sensitive-files
         echo "Preparing the benchmark results:"
-        tar -cvf benchmark-results-${{ github.event.inputs.jdk }}-${{ github.event.inputs.scala }}.tar `git diff --name-only` `git ls-files --others --exclude=tpcds-sf-1 --exclude-standard`
+        tar -cvf benchmark-results-${{ github.event.inputs.jdk }}-${{ github.event.inputs.scala }}.tar `git diff --name-only` `git ls-files --others --exclude=tpcds-sf-1 --exclude=tpch-sf-1 --exclude-standard`
```

Comment:
@srinivasst  I remove `--exclude=tpch-sf-1` and download the generated tpch data, then I found that the data file only contains parquet footer and no data

Reply:
Interesting find, let me check the local data, but do you know where else would tpch dbgen be available from?
I'm using from github.com/databricks/tpch-dbgen

Reply:
Sorry, I don’t know ...